core/lib/generators/solidus/update/update_generator.rb

```diff
@@ -29,6 +38,12 @@ class UpdateGenerator < ::Rails::Generators::Base
                  hide: true
 
     def create_new_defaults_initializer
+      previous_version_prompt = options[:previous_version_prompt]
+      return if previous_version_prompt && !yes?(<<~MSG, :red)
```

Comment:
I'm not sure I understand why this is needed. Shouldn't this work with any Solidus version?

Reply:
Yes, it should work with any Solidus version. However, we agreed with @kennyadsl that we should only encourage upgrading from the last minor version. On top of that, it allows us to assume the `from` version so that users don't need to provide it manually, which is not too ergonomic.

Reply:
@waiting-for-dev okay, understood! In that case, should we make it clear that you can still upgrade from an earlier version, but it's not recommended? With the current messaging, it looks like the process won't work at all if you upgrade from an earlier version.

Reply:
I updated the message and removed the `hide: true` option from the `from` argument. Probably you're right. Still, I can't help feeling a bit uncomfortable :slightly_smiling_face: